### ISSUE TITLE
UX Feature: Make Zap resizable and borderless with a title bar

### ISSUE THREAD

**Issue by thinkjanis** [LN-Zap/zap-desktop#131]
Most apps are not fullscreen so the same should be applied to Zap. If there is a desire to have this option, then a toggle should be added in settings.

Minimal size for the app (min-width and min-height) should be defined.
![zap](https://user-images.githubusercontent.com/31632325/34395181-0549668c-eb67-11e7-84d3-cb98118b73c3.png)

**Comment by thinkjanis** [comment]
Minimal width/height should be a bit smaller so that the wallet can have some space around it on a 1280×720 screen. Some CSS optimizations will be needed.

```let icon = path.join(__dirname, '..', 'resources', 'icon.icns')
  mainWindow = new BrowserWindow({
    frame: true,
    minWidth: 1100,
    minHeight: 700,
  })

  mainWindow.setMenu(null)
  mainWindow.setAutoHideMenuBar(true)

**Comment by JimmyMow** [comment]
@thinkjanis ooo that's a cool screenshot. 

yeah this shouldn't be too difficult. i'll assign this to myself.

**Comment by JimmyMow** [comment]
fixed #182
